src/edit/handler/Edit.Poly.js

```diff
@@ -70,6 +70,10 @@ L.Edit.Poly = L.Handler.extend({
     },
 
     _initMarkers: function () {
+
+        var poly = this._poly;
+        var maxPoints = poly.options.maxPoints || 0;
```

Comment:
I feel like this makes more sense as an edit option rather than a polygon option. That's how I did the icons at least. Maybe we should get a third opinion

Reply:
not sure what the editor component is, but it was relatively straightforward to implement this on our side when passing down the options for the polygon see here: https://github.com/dronedeploy/angular2/pull/2988

so for that reason, i vouch for it.

Reply:
There's an options dictionary that gets passed into the constructor of this class (see line 8 of this file). You can set default options there as well. Personally thinks it makes more sense to go there since that option only affects the draw code, might be worth getting opinion of someone on the flight team.